lambda-layers/terraform/README.md

```diff
@@ -0,0 +1,49 @@
+# Lambda Layers with Terraform
+
+Simple demo application illustrating Lambda layers using LocalStack, deployed via Terraform.
+
+## Prerequisites
+
+* LocalStack
+* Docker
+* Terraform & `tflocal`
+* Python & `pip`
+* `make`
+
+## Installing
+
+To install the dependencies:
+```
+make install
+```
+
+## Running
+
+Make sure that LocalStack is started:
+
+```
+LOCALSTACK_AUTH_TOKEN=... DEBUG=1 localstack start
```

Comment:
General Idea for docs: Would it make sense to use a variable such as `$LOCALSTACK_AUTH_TOKEN` to facilitate copy/pasting the command rather using some bash-incompatible placeholder (e.g., `...` or `<>`)?